### ISSUE TITLE
potentially work on making it 0 dependencies like `per-env`?

### ISSUE THREAD

**Issue by ForsakenHarmony** [shian15810/by-node-env#121]
I gotta be honest it's completely fucking insane to me that this expands to 72 dependencies, the npm ecosystem is a mess

https://npm.anvaka.com/#/view/2d/by-node-env

I know it's maybe not a priority or even seems like an issue to you, so feel free to close it

**Comment by shian15810** [comment]
@ForsakenHarmony Yeah sure, I would like to work on that. Give me some time.

**Comment by ForsakenHarmony** [comment]
with dotenv support I guess 0 is not really viable

**Comment by shian15810** [comment]
Yeah, I think the goal should be just keeping `dotenv` and its dependencies.
